main.go

```diff
@@ -1,24 +1,29 @@
 package main
 
 import (
+	"context"
 	"database/sql"
+	"fmt"
 	"os"
+	"os/signal"
 	"strconv"
+	"syscall"
 
 	"github.com/ONSdigital/dp-filter-api/api"
 	"github.com/ONSdigital/dp-filter-api/config"
 	"github.com/ONSdigital/dp-filter-api/filterJobQueue"
 	"github.com/ONSdigital/dp-filter-api/postgres"
 	"github.com/ONSdigital/go-ns/kafka"
 	"github.com/ONSdigital/go-ns/log"
-	"github.com/ONSdigital/go-ns/server"
-	"github.com/gorilla/mux"
 	_ "github.com/lib/pq"
 )
 
 func main() {
 	log.Namespace = "dp-filter-api"
 
+	signals := make(chan os.Signal)
```

Comment:
is it worth giving this a buffer of 1? I know its unlikely but if a signal is sent before exec reaches the select block it will block the app. (Sorry I am being picky)